LWCOSC/force-app/main/default/classes/ProductSelectorImpl.cls

```diff
@@ -0,0 +1,45 @@
+public with sharing class ProductSelectorImpl extends fflib_SObjectSelector implements IProductSelector  {
```

Comment:
Selectors should not have "with sharing" clause, let the caller decide the mode of sharing. It however should be "inherited sharing"

Reply:
Done